src/Microsoft.SqlTools.ServiceLayer/Formatter/TSqlFormatterService.cs

```diff
@@ -13,6 +13,7 @@
 using Microsoft.SqlTools.ServiceLayer.Formatter.Contracts;
 using Microsoft.SqlTools.ServiceLayer.Hosting;
 using Microsoft.SqlTools.ServiceLayer.Hosting.Protocol;
+using Microsoft.SqlTools.ServiceLayer.LanguageServices;
```

Comment:
You're right - we should do this. However I'm not going to get that into the codebase in time for 0.3.0 release. Therefore I've created https://github.com/Microsoft/vscode-mssql/issues/712 to track this - we should tackle when we can.